server/teams.js

```diff
@@ -216,7 +228,7 @@ module.exports = function(imports) {
 		} catch (err) {
 			res.status(500).end("failed parsing TBA json");
 		}
-		
+
```

Comment:
If the team does not exist on TheBlueAlliance, it should give a 404 instead of 500 error. In this case, TheBlueAlliance returns a status code of 404.